packages/core/src/lib/tooltip.spec.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
```

Comment:
Do you mind adding a lil docstring explaining the need for an extra test-only component?